index.html

```diff
@@ -1 +1,402 @@
-TODO
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+	  <meta name="description" content="Ember Test Selectors - enabling better element selectors in Ember.js tests"/>
+  	<meta name="keywords" content="emberjs, javascript, tests, selectors, data-test" />
+  	<meta name="language" content="en"/>
+  	<meta name="content-language" content="en"/>
+  	<meta name="publisher" content="simplabs - Marco Otte-Witte" />
+    <title>Ember Test Selectors</title>
+    <!--<link rel="shortcut icon" type="image/x-icon" href="images/favicon.ico" />-->
+    <link rel="apple-touch-icon" href="images/apple-57.png">
+    <link rel="apple-touch-icon" sizes="72x72" href="images/apple-72.png">
+    <link rel="apple-touch-icon" sizes="114x114" href="images/apple-114.png">
+    <link rel="apple-touch-icon" sizes="144x144" href="images/apple-144.png">
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/3.1.1/css/font-awesome.css">
+    <link rel="stylesheet" href="https://yandex.st/highlightjs/7.4/styles/github.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Ubuntu:300,400">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+    <style type="text/css" media="screen">
+      body {
+        line-height: 1.7;
+        font-weight: 300;
+        color: #384147;
+      }
+    
+      a, a:visited {
+        color: #99FFCC;
```

Comment:
the link text is a bit hard to read in this color - we might have to add a background color, e.g. black.